### DISCUSSION TITLE
Octoherd on github enterprise

### DISCUSSION THREAD

**Discussion by TFA5KOR** [octoherd/octoherd#36] [Q&A] [answered]
By default octoherd cli looks for the repo on "api.github.com" but I am trying to use Octoherd script for the org on Github enterprise. Is there a way to change the baseurl for the Octokit connection being setup on Octoherd CLI? 
Or what will be the best way to achieve this?

**Comment by gr2m** [comment, marked as answer]
see my answer at https://github.com/octoherd/octoherd/issues/35#issuecomment-1198233581

> As Octoherd is using [`@octokit/core`](https://github.com/octokit/core.js/) internally, adding support for GHES should be possible, I just didn't run into it myself. I think in order to make scripts run with GHES, we'd have to add a `base-url` option to [the Octoherd CLI](https://github.com/octoherd/cli). Compare Octokit's `baseUrl` option

<img width="942" alt="image" src="https://user-images.githubusercontent.com/39992/181551629-08b6fd7c-f831-4b2f-8e79-2935ce5d6fb0.png">
